tsconfig.json

```diff
@@ -1,21 +1,20 @@
 {
   "include": [
     "./src/**/*",
-    "./node_modules/web3-typescript-typings/index.d.ts",
     "./node_modules/types-bn/index.d.ts",
     "./node_modules/types-ethereumjs-util/index.d.ts"
   ],
   "compilerOptions": {
     "outDir": "lib",
-    "target": "es5",
+    "target": "es6",
     "lib": ["dom", "esnext"],
     "allowSyntheticDefaultImports": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noErrorTruncation": true,
     "noImplicitThis": false,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Can we keep this on `true`. Unused locals make no sense, and we should just out remove them where applicable.

Reply:
Adding a comment that it's needed due to auto generated files